lib/graphql/utils.ex

```diff
@@ -71,6 +71,8 @@ defmodule BridgeEx.Graphql.Utils do
 
   @spec normalize_inner_fields(%{atom() => any()} | String.t()) :: %{atom() => any()} | String.t()
   def normalize_inner_fields(binary) when is_binary(binary), do: binary
+  def normalize_inner_fields(number) when is_number(number), do: number
```

Comment:
At this point do we need to check if it's binary, number or boolean: why not just return something if it's not a map? (I don't know the full implication of this)

Reply:
agree